src/orca-log-catcher/ErrorBoundary.res

```diff
@@ -157,6 +157,7 @@ module ErrorCard = {
           firstEvent: false,
           metadata: JSON.Encode.null,
           ephemeralKey: "",
+          componentName,
```

Comment:
@Sanskar2001 will we be able to see this new type directly in Grafana or some other changes will be required? The use of this type is to help detect the component which crashed

Reply:
@PritishBudhiraja We don't want to introduce one more column in sdk_events for this, can you please append the component name in value only?

Reply:
updated please check once